### PR TITLE
[CHORE] E722 bare exception fix

### PR DESCRIPTION
**Pull request by dni**
remove all bare exceptions from codebase and change it in `.flake8`

Bare except will catch exceptions you almost certainly don't want to catch, including KeyboardInterrupt (the user hitting Ctrl+C) and Python-raised errors like SystemExit

If you don't have a specific exception you're expecting, at least except Exception, which is the base type for all "Regular" exceptions.

https://stackoverflow.com/questions/54948548/what-is-wrong-with-using-a-bare-except

### CODE REVIEW

**Comment by motorina0** [comment]
@dni please check if there are other large PRs pending (c-lightning rest?). This PR might cause a lot of conflicts

**Comment by dni** [comment]
pretty hard to check, before it merged, but i can easily solve all conflicts in PRs.

File: lnbits/decorators.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ async def get_key_type(
                 pass
             else:
                 raise
-        except:
+        except Exception:
             raise
```
Comment (motorina0) [inline lnbits/decorators.py:177]: - isn't this `except` redundant?

**Review by motorina0 — APPROVED** [review]
LGTM. 
See the two open questions.

File: lnbits/decorators.py (hunk 1 above)
Comment (dni) [reply lnbits/decorators.py:177]: Bare except will catch exceptions you almost certainly don't want to catch, including KeyboardInterrupt (the user hitting Ctrl+C) and Python-raised errors like SystemExit

If you don't have a specific exception you're expecting, at least except Exception, which is the base type for all "Regular" exceptions.

https://stackoverflow.com/questions/54948548/what-is-wrong-with-using-a-bare-except

File: lnbits/app.py [hunk 2]
```diff
@@ -133,7 +133,7 @@ def signal_handler(signal, frame):
                 f"The backend for {WALLET.__class__.__name__} isn't working properly: '{error_message}'",
                 RuntimeWarning,
             )
-        except:
+        except Exception:
```
[the remark this answers is not in the document]
Comment (dni) [reply lnbits/app.py:136]: check descripition

File: lnbits/decorators.py (hunk 1 above)
Comment (dni) [reply lnbits/decorators.py:177]: looking at it the whole thing is suspicious. i would defer it to another PR

**Review by talvasconcelos — APPROVED** [review]

**Pull request merged**